docs/guide.rst

```diff
@@ -196,31 +196,116 @@ the scene as desired::
 
 .. image:: _static/guide_rotating_cube.gif
 
+
 World objects
 -------------
 
+We've briefly mentioned world objects, materials, and geometry. But how do these relate?
+
 A world object represents an object in the world. It has a transform, by which the
 object can be positioned (translated, rotated, and scaled), and has a visibility property.
 These properties apply to the object itself as well as its children (and their children, etc.).
 
+All objects that have an appearance in the scene are world objects. But there
+are also helper objects, lights, and cameras. These are all world objects.
 
-Geometry
---------
+**Geometry**
 
-Each world object has a geometry. This geometry object contains the
+Most world objects have a geometry. This geometry object contains the
 data that defines (the shape of) the object, such as positions, plus
 data associated with these positions (normals, texcoords, colors, etc.).
 Multiple world objects may share a geometry.
 
 
-Materials
----------
+**Materials**
 
-Each world object also has a material. This material object defines the
+All visible world objects have a material. This material object defines the
 appearance of the object. Examples can be its color, how it behaves under lighting,
 what render-mode is applied, etc. Multiple world objects may share a material.
 
 
+Cameras and controllers
+-----------------------
+
+We've already been using cameras, but let's look at them a bit closer!
+
+
+**Perspective camera**
+
+There are two main cameras of interest. The first is the :class:`~pygfx.cameras.PerspectiveCamera`,
+which is a generic camera intended for 3D content. You can instantiate one
+like this:
+
+.. code-block:: python
+
+    camera = gfx.PerspectiveCamera(50, 4/3)
+
+The first argument is the fov (field of view) in degrees. This is a value between 0 and 179,
+with typical values < 100.
+
+The second argument is the aspect ratio. A window on screen is usually a rectangle
+with 4/3 or 16/9 aspect. The aspect can be set so the contents better fit the window.
+
+When the fov is zero, the camera operates in orthographic mode, where there is
+no perspective. In this case it'd be good to also specify a 3d argument: the extent
+specifies the size of the thing being viewed. So if your scene is about 400 units in size:
+
+.. code-block:: python
+
+    camera = gfx.PerspectiveCamera(0, 1, 400)
```

Comment:
I don't understand. 400 is the size? In what dimension? Depth? Width? Diagonal?

Reply:
Width and height, depending on the aspect.

Reply:
Can you give an example? How would I determine a proper value for extent, as a user?